readme.md

```diff
@@ -75,6 +75,10 @@ Returns the previous item.
 
 Returns the item by going the given amount of `steps` through the array. For example, calling `step(2)` is like calling `next()` twice. You go backward by specifying a negative number.
 
+#### peek(steps)
+
+Returns the item that is located in the given amount of `steps` through the array. For example, calling `peek(2)` would get the item 2 items after the current one. You go backward by specifying a negative number.
```

Comment:
I think this should more explicitly say how it's different from `step`.

Reply:
Fixed! (For some reason, I'm not seeing the commit propagate in the PR but it does appear when [viewing the entire branch](https://github.com/Richienb/cycled/tree/add-peek))